main.go

```diff
@@ -61,6 +61,7 @@ func main() {
 	audience := flag.String("token-audience", "sts.amazonaws.com", "The default audience for tokens. Can be overridden by annotation")
 	mountPath := flag.String("token-mount-path", "/var/run/secrets/eks.amazonaws.com/serviceaccount", "The path to mount tokens")
 	tokenExpiration := flag.Int64("token-expiration", 86400, "The token expiration")
+	region := flag.String("aws-default-region", "", "The region the cluster is running in. We use this value to set AWS_DEFAULT_REGION in mutated containers")
```

Comment:
nit: Can you remove the first sentence? This is not necessarily the region the _cluster_ is running in, it is the value that is injected into a a container. And also call out that an unset value will not inject anything